src/components/LocaleSelector/LocaleSelector.jsx

```diff
@@ -1,18 +1,30 @@
 import PropTypes, { func, string } from 'prop-types'
 import React from 'react'
+import classnames from 'classnames'
 import { Link } from '@material-ui/core'
 
-const LocaleSelector = ({ locales }) =>
-  locales.map(({ icon: Icon, name, url }) => (
-    <Link href={url} key={name} title={name.toUpperCase()}>
-      <Icon alt={name} />
-    </Link>
-  ))
+import './index.scss'
+
+const LocaleSelector = ({ locales, active }) => (
```

Comment:
I've recently merged changes to this component, so there might be conflicts in rebase